be/src/olap/rowset/segment_v2/bitshuffle_wrapper.cpp

```diff
@@ -34,6 +34,15 @@
 #undef bshuf_compress_lz4
 #undef bshuf_decompress_lz4
 
+#undef BITSHUFFLE_H
+#define bshuf_compress_lz4_bound bshuf_compress_lz4_bound_avx512
+#define bshuf_compress_lz4 bshuf_compress_lz4_avx512
+#define bshuf_decompress_lz4 bshuf_decompress_lz4_avx512
+#include <bitshuffle/bitshuffle.h> // NOLINT(*)
+#undef bshuf_compress_lz4_bound
+#undef bshuf_compress_lz4
+#undef bshuf_decompress_lz4
+
```

Comment:
Hi @HackToday , these lines look tricky. Why should we do this?

Reply:
it is for two versions (avx2 and avx512) to make it run selectively according to platform features.
It was old tricky (avx2 vs non-avx2), is it ?

Reply:
> it is for two versions (avx2 and avx512) to make it run selectively according to platform features. It was old tricky (avx2 vs non-avx2), is it ?

Okay, I see. I think you would better add comments like [L26](https://github.com/apache/doris/blob/ce4c8a710de456c9e919817ab7e8b59086a3726c/be/src/olap/rowset/segment_v2/bitshuffle_wrapper.cpp#L26).

Reply:
Have resolved as comment

Reply:
Okay.